CGPs/0031.md

```diff
@@ -0,0 +1,63 @@
+# CGP [0031]: Granda Mento
+
+- Date: 2020-06-15
+- Author(s): @tkporter, @albertclabs
+- Status: DRAFT
+- Governance Proposal ID #: 31
+- Date Executed: [if executed]
+
+## Overview
```

Comment:
Please reference the specification CIP-38 in Overview section and how this proposal will activate any of the parameters specified in CIP-38. At the moment it feels like two separate documents and it will lead to confusion. 

CGPs are just activation methods for CIP specifications, the same way a Hardfork is. The CIP is the source of truth for any core protocol changes.